src/views/Home.vue

```diff
@@ -166,6 +166,19 @@
             this.afterFilter();
         }
 
+        @Watch("tagsModel.sorting")
```

Comment:
Could we pass the sorting prop to the `AntiPatternsContainerComponent.vue` since this doesn't affect the number of antipatterns but only its arrangment. We already have the displayed list of antipatterns in this component as well.